integration/terraform/ec2/README.md

```diff
@@ -186,6 +232,8 @@ secrets.
 3. Select the `Run Integration Tests` action
 4. Select `Run workflow`, and choose the branch to execute integration tests on
 
+> Note: The Amazon EC2 quota limit might need to be increased to run the integration test: request an increase via `All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Request`
```

Comment:
Ideally the integration tests shouldn't need this quota increase and should be able to queue things and execute as instance capacity frees up, but that's a problem for another day. As part of this note, can you call out that each EC2 integration test suite gets run on a different EC2, per OS tested?

Reply:
Updated the note as suggested. Thanks!